content/tutorial/tutorial.md

```diff
@@ -440,7 +440,7 @@ class Square extends React.Component {
     return (
       <button
         className="square"
-        onClick={() => this.props.onClick()}
+        onClick={() => this.props.onClick}
```

Comment:
This wouldn't call the `onClick` prop anymore when a user clicks on the `button`.

Does this clear things up?

Reply:
I closed the issue. Actually, I realized I had no callback function. It was actually confusing because I was passing down props from the board to the square so automatically I thought we'll access the function directly without a callback. Found it a couple of chapters later in the tutorial and decided to close the issue.
Thanks anyway @eps1lon